source/install/install-mmte-helm-gitlab-helm.rst

```diff
@@ -1,38 +1,67 @@
 ..  _install-mmte-helm-gitlab-helm:
 
-How to install Mattermost Team Edition Helm Chart in GitLab Helm Chart deployment
+How to install Mattermost Team Edition Helm Chart in a GitLab Helm Chart deployment
 =====================================================================================
 
-This document describes how to install Mattermost Team Edition Helm Chart in proximity with an existing GitLab Helm Chart deployment, utilizing shared configurations to streamline authentication, storage, encryption, and traffic routing.
+This document describes how to use Mattermost Team Edition Helm Chart in proximity with an existing GitLab Helm Chart deployment. Once the Mattermost Team Edition Helm Chart is installed, GitLab SSO integration is configured which utilizes shared configurations to streamline authentication, storage, encryption, and traffic routing.
 
 As the Mattermost Helm Chart is installed in a separate namespace, it is recommended that cert-manager and nginx-ingress be configured to manage cluster-wide ingress and certificate resources. 
 
-The steps in this document presume in-chart Minio instance usage. For information about out-of-chart object storage configuration, review `this document <https://gitlab.com/gitlab-org/charts/gitlab/tree/master/doc/charts/registry#storage>`__ for GCS and S3 examples. Alternatively, visit your provider's Help documentation for configuration settings. 
-
 
 Prerequisites
 ----------------------------
 
-  - A running Kubernetes cluster with a PostgreSQL database.
-  - A running GitLab Helm Chart release.
-  - The name of the secret that holds your PostgreSQL password ``<gitlab>-postgresql-password``.
-  - The name of the secret that holds your Minio keys ``<gitlab>-minio-secret``.
-  - The service name for your PostgreSQL, ``<gitlab>-postgresql``, and the port. If you installed the GitLab helm chart in ``default`` namespace, then the port is ``5432``.
-  - The service name for Minio and the port, ``<gitlab>-minio-svc``, and the port. If you installed the GitLab helm chart in ``default`` namespace, then the port is ``9000``.
-  - The names of ``kubernetes.io/ingress.class``, ``kubernetes.io/ingress.provider`` and ``certmanager.k8s.io/issuer``.
-  
+  - A running Kubernetes cluster.
+  - Helm v2.
+  - `Tiller <https://rancher.com/docs/rancher/v2.x/en/installation/ha/helm-init/>`_ (the Helm server-side component)     installed on the cluster.
+
 **Note:**
-There is an option to use external database services (PostgreSQL or MySQL) for your Mattermost installation. If you use an external database you will need to disable the MySQL chart in the ``values.yaml`` as per the example in this document.
+For the Team Edition you can have just one replica running.
```

Comment:
Why do we mention this here?

Reply:
It's from https://github.com/mattermost/mattermost-helm/tree/master/charts/mattermost-team-edition listed as a limitation. I thought it may have an impact on the deployment or let the user know upfront that TE might not be the best option for their use case so I kept it in. 0/5.

Reply:
Okay, it seemed to stand out and wasn't clear why it was included there. But 0/5